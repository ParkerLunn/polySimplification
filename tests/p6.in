x^2*(x*(3x^3 + 4x^4))
