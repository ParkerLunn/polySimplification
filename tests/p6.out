((x)^2*(x*((3*(x)^3)+(4*(x)^4))))
4x^7 + 3x^6